test/argparse_test3.jl

```diff
@@ -80,7 +80,7 @@ let s = ap_settings3()
           -k                    provide the answer
           -u                    provide the answer as floating point
           --array ARRAY         create an array (type: Array{$Int,1},
-                                default: [7,3,2])
+                                default: [7, 3, 2])
```

Comment:
The problem here is that `string help(s)` puts spaces in between in case of v0.6 and doesn't for v0.5. Trying to find out why.